observatory-platform/observatory/platform/cli/templates/generate_project/setup.cfg.jinja2

```diff
@@ -0,0 +1,50 @@
+{# Copyright 2020 Curtin University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Author: Aniek Roelofs -#}
+[metadata]
+name = {{ package_name }}
+python-requires = >=3.7
+license = Apache License Version 2.0
+classifier =
+    Development Status :: 2 - Pre-Alpha
+    Environment :: Console
+    Environment :: Web Environment
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: Apache Software License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Topic :: Scientific/Engineering
+    Topic :: Software Development :: Libraries
+    Topic :: Software Development :: Libraries :: Python Modules
+    Topic :: Utilities
+keywords =
+    science
+    data
+    workflows
+
+[files]
+packages =
+    {{ package_name }}
+
+[extras]
+tests =
+    liccheck==0.4.*
```

Comment:
These package versions should be templated out.

Reply:
How would you determine the package versions?
I've more or less just copied to setup.cfg from our observatory-dags, but I'm thinking whether we need these extras at all.
There's nothing in the generated workflow project/files that necessarily require them I think. Maybe people could add them themselves after generating the project if they want?

Reply:
Ok no problem.  Actually don't worry about my comment.  If you do decide to leave them in, I can template them out later as part of other work to template out version requirements.